cypress/integration/select-one.spec.js

```diff
@@ -861,5 +861,39 @@ describe('Choices - select one', () => {
           });
       });
     });
+
+    describe('new, destroy, init', () => {
+      it('has 3 choices', () => {
+        cy.get('[data-test-hook=new-destroy-init]')
+          .find('.choices__list--dropdown .choices__list')
+          .children()
+          .should('have.length', 3)
+      });
+
+      it('has 3 choices after destroy', () => {
```

Comment:
This should have a describe around it with a description like `destroying the choices instance` and then the `it` description should be along the lines of `the original select element is preserved`

Reply:
Is a0626507edee0a8db4a2d5a16ffdf65df7ac6463 your expected one...?